user.go

```diff
@@ -24,3 +27,21 @@ func (u *User) String() string {
 func (u *User) Mention() string {
 	return fmt.Sprintf("<@%s>", u.ID)
 }
+
+// AvatarURL returns a URL to the user's avatar.
+//		size:     The size of the user's avatar as a power of two
+func (u *User) AvatarURL(size string) string {
+	return UserAvatarURL(u.ID, u.Avatar, size)
+}
+
+// UserAvatarURL returns a URL to the requested user's avatar
+//		userID:   The ID of the user to get
+//		avatarID: The ID of the user's avatar
+//		size:     The size of the user's avatar as a power of two
+func UserAvatarURL(userID, avatarID, size string) string {
+	extension := ".jpg"
+	if strings.HasPrefix(avatarID, "a_") {
+		extension = ".gif"
+	}
+	return EndpointCDNAvatars + userID + "/" + avatarID + extension + "?size=" + size
```

Comment:
Hmm, I would prefer:
```
func (u *User) AvatarURL(size string) string {
  var url string
  if strings.HasPrefix(avatarID, "a_) {
    url = EndpointUserAvatarAnimated(userID, avatarID)
  } else {
    url = EndpointUserAvatar(userID, avatarID)
  }
  return url + "?size=" + size
}
```

We already support EndpointUserAvatar, adding an animated one seems reasonable.